tests/Php84/Php84Test.php

```diff
@@ -319,4 +329,9 @@ public static function mbRTrimProvider(): iterable
 
         yield ["foo\n", "foo\n", 'o'];
     }
+
+    public static function bcDivModProvider(): iterable
+    {
+        yield ['10', '10', null, ['1', '0']];
```

Comment:
A couple more tests would be nice. You're only testing the simplest path.

Also, beware that by setting the scale to `null`, your test depends on the global `bcmath.scale` setting. Currently, the test will fail if it's run in an evironment wherre that setting is not `0`.